src/File/Strategy/Local.php

```
@@ -8,7 +8,7 @@
 
 class Local extends AbstractStrategy implements StrategyInterface
 {
-    public function downloadFile($fileInfo, $destinationPath)
+    public function downloadFile($fileInfo, $destinationPath, $overwrite)
```

just a note for future if we want ti implement this in `Local` we'll need to update the `writeFile` somehow.  Currently `dumpFile` has hard overwrite `true` 
https://github.com/symfony/symfony/blob/35dbf8c81fad376da61978d7ffd48b623522b97a/src/Symfony/Component/Filesystem/Filesystem.php#L668
Obviously that doesn't matter for jobs, but sandboxes may want that flexibility

---

Yes, I was wondering whether to make this complete or not, but then we should make that option for databases workspaces too and that seemed to be a long journey.

---

yeah, I think we can put it off.  maybe we should throw a user exception if anyone tries `overwrite: false` on local?

---

good idea, done that, so that it's not ending in vacuum